packages/backend-defaults/src/alpha/entrypoints/metrics/DefaultMetricsService.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  MetricsService,
+  ObservableMetric,
```

Comment:
Is there any particular reason that `ObservableMetric` became a local type (rather than taking something from `@opentelemetry/api`), unlike the others?

Reply:
Yes and the idea of providing a simpler interface, but I am starting to go back on this decision. I think we use the type coming from otel directly and iterate on it as needed. Doing that will get this out in `alpha` much faster.